redisson/src/main/java/org/redisson/RedissonListMultimap.java

```diff
@@ -328,30 +328,50 @@ public List<V> replaceValues(K key, Iterable<? extends V> values) {
         return (List<V>) get(replaceValuesAsync(key, values));
     }
 
+    @Override
+    public void fastReplaceValues(final K key, final Iterable<? extends V> values) {
+        get(fastReplaceValuesAsync(key, values));
+    }
+
     @Override
     public RFuture<Collection<V>> replaceValuesAsync(K key, Iterable<? extends V> values) {
-        List<Object> params = new ArrayList<Object>();
+        return replaceValuesInternal(key, values, true);
+    }
+
+    @Override
+    public RFuture<Void> fastReplaceValuesAsync(K key, Iterable<? extends V> values) {
+        return replaceValuesInternal(key, values, false);
+    }
+
+    private <T> RFuture<T> replaceValuesInternal(K key, Iterable<? extends V> values, boolean returnValues) {
```

Comment:
A separate method is needed without returnValues parameter and "local members = redis.call('lrange', KEYS[2], 0, -1);" usage

Reply:
Yes, please make separate method.

Reply:
Done